### PR TITLE
Update Gradle to 7.4.1

### PR DESCRIPTION
**Pull request by wraschke**
#build

### CODE REVIEW

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
Ethan's original PR is https://github.com/OpenLiberty/open-liberty/pull/19808. I didn't want to push my commits to his fork so why I have this PR; his commits are still there.

**Comment by wraschke** [comment]
#ol-pbbeta

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#ol-pbbeta

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
Hi, Ethan!

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#fullpbbeta

**Comment by wraschke** [comment]
#pb #build

**Comment by wraschke** [comment]
#fullpb

**Comment by wraschke** [comment]
#fullpbbeta

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#pb

**Comment by wraschke** [comment]
#build #pb

**Comment by wraschke** [comment]
#pbbeta

**Comment by wraschke** [comment]
#pb #build

**Comment by wraschke** [comment]
For reference: https://github.com/OpenLiberty/open-liberty/pull/24089

**Comment by wraschke** [comment]
#pb #build

**Comment by wraschke** [comment]
#pb

**Comment by wraschke** [comment]
#pb

**Comment by wraschke** [comment]
#pb-beta

**Comment by wraschke** [comment]
#pbbeta

**Comment by wraschke** [comment]
#pb

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#pbbeta@dev

**Comment by wraschke** [comment]
#pbbeta@dev

**Comment by wraschke** [comment]
#pb #build

**Comment by wraschke** [comment]
#pb #build

**Comment by wraschke** [comment]
#pbbeta@dev

**Comment by wraschke** [comment]
#pb@dev

File: .ci-orchestrator/scheduledPersonalBuilds.yml [hunk 1]
```diff
@@ -228,8 +228,8 @@ steps:
     fat.test.mode: lite
     fats_to_omit: "com.ibm.ws.collective.controller.deploy_fat, com.ibm.ws.health.manager.odrlib_fat, com.ibm.ws.dynamic.routing_ihs_fat, com.ibm.ws.node.scaling_fat, com.ibm.ws.scaling.member_fat_multinode, com.ibm.ws.node.health_fat"
     fat_uploads_to_expect: ${compile:fat_uploads_to_expect}
-    fileserver: libfsfe04.hursley.ibm.com
-    outputServer: libfsfe04.hursley.ibm.com
+    fileserver: libfsfe0504.hursley.ibm.com
```
Comment (fritze2) [inline .ci-orchestrator/scheduledPersonalBuilds.yml]: `libfsfe0504.hursley.ibm.com` should be `libfsfe05.hursley.ibm.com`

File: .ci-orchestrator/scheduledPersonalBuilds.yml [hunk 2]
```diff
@@ -228,8 +228,8 @@ steps:
...
+    outputServer: libfsfe0504.hursley.ibm.com
```
Comment (fritze2) [inline .ci-orchestrator/scheduledPersonalBuilds.yml]: Same as above.

**Comment by wraschke** [comment]
The https://wasrtc.hursley.ibm.com:9443/jazz/resource/itemOid/com.ibm.team.build.BuildResult/_ZsjqMKf2Ee24er7gxU0d3Q build was good and had only one known failure--but now it's gone!

I will request another one.

Please ignore all the pipeline links. I am using this PR to test other, non-related functionality.

Additionally, I did a manual comparison locally using Beyond Compare between an Open Liberty image built from the `release` branch and then this PR's feature branch, rebased on `release`. i.e., gradle7's commits sit on the stack of same commits in `release`. The only differences are in the `build.image/wlp/lib/platform/checksums/` files and that's to be expected, I believe.

#build

File: .ci-orchestrator/scheduledPersonalBuilds.yml (hunk 1 above)
Comment (crpotter) [inline .ci-orchestrator/scheduledPersonalBuilds.yml]: Is this supposed to just be `libfsfe05.hursley.ibm.com` like all the other changes?

File: dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CDITests.java [hunk 3]
```diff
@@ -31,15 +30,12 @@
...
-    private static final String TEST_APP_15_WITH_CDI = "testSpringBootApp15WithCDIFeatureEnabled";
     private static final String TEST_APP_20_WITH_CDI = "testSpringBootApp20WithCDIFeatureEnabled";
 
     @Override
     public String getApplication() {
         String methodName = testName.getMethodName();
-        if (TEST_APP_15_WITH_CDI.equals(methodName)) {
-            return SPRING_BOOT_15_APP_BASE;
-        } else if (TEST_APP_20_WITH_CDI.equals(methodName)) {
+        if (TEST_APP_20_WITH_CDI.equals(methodName)) {
```
Comment (crpotter) [inline dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CDITests.java:38]: Do we need developers to check over actual FAT test changes?

File: dev/settings.gradle [hunk 4]
```diff
@@ -187,6 +189,9 @@ if (!JavaVersion.current().isJava11Compatible()) {
 
 }
 
+//DirectoryScanner.removeDefaultExclude("**/.gitignore")
```
Comment (crpotter) [inline dev/settings.gradle:192]: Are these being left in for future use? If they are, should a comment be left for them? If not, should these lines be removed?

File: .ci-orchestrator/scheduledPersonalBuilds.yml (hunk 1 above)
Comment (wraschke) [reply .ci-orchestrator/scheduledPersonalBuilds.yml]: That's an error I will fix.

**Review by crpotter — DISMISSED** [review]

File: dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CDITests.java (hunk 3 above)
Comment (wraschke) [reply dev/com.ibm.ws.springboot.support_fat/fat/src/com/ibm/ws/springboot/support/fat/CDITests.java:38]: This is from commit 068c82f8a3e213c6ec3a3645d074047c6d8cad64, which Anjum from OSGi apps team did. Her team is the SME on Spring Boot tests and etc.

File: dev/settings.gradle (hunk 4 above)
Comment (wraschke) [reply dev/settings.gradle:192]: Sorry, I don't know the answer to this. This is a change that Ethan had done.

**Comment by wraschke** [comment]
#build

**Comment by wraschke** [comment]
#libby

**Review by fritze2 — APPROVED** [review]

**Comment by wraschke** [comment]
The PB with https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_7ieIYK1fEe2FnaZGnlWrhQ mentioned in LibbyBot's [comment right above](https://github.com/OpenLiberty/open-liberty/pull/23350#issuecomment-1432415246) failed not due to this PR's changes but because of the `/liberty/personal/1` partition on libertyfs reaching maximum capacity again. The PB with UUID ZsjqMKf2Ee24er7gxU0d3Q was good with one known failure and has been deleted, as mentioned earlier. And again, the Beyond Comparison verification was good.

And additionally, the `.ci-orchestrator/scheduledPersonalBuilds.yml` corrected file I committed and pushed yesterday has no bearing on product, tests, built binaries, any build verification activities, etc. It's used for CI Orchestrator pipelines only.

I asked Ethan if he was okay with this merging and he gave me his approval. Ethan will keep an eye out for issues in LAOS continuous builds that might be related to this PR.

**Comment by wraschke** [comment]
This is too close to the 23.0.0.2 GM driver cutting date (February 21) so I am reverting.

**Pull request merged**